mod.ts

```diff
@@ -1,65 +1,25 @@
-export { get, put, patch, del, post } from "./source/methods.ts";
-export { filterStatus, filterBody, filterResponse } from "./source/filters.ts";
-export {
-  CONTINUE,
-  SWITCHING_PROTOCOL,
-  PROCESSING,
-  EARLY_HINTS,
-  OK,
-  CREATED,
-  ACCEPTED,
-  NON_AUTHORATIVE_INFORMATION,
-  NO_CONTENT,
-  RESET_CONNECTION,
-  PARTIAL_CONTENT,
-  MULTI_STATUS,
-  ALREADY_REPORTED,
-  IM_USED,
-  MOVED_PERMANENTLY,
-  FOUND,
-  SEE_OTHER,
-  NOT_MODIFIED,
-  TEMPORARY_REDIRECT,
-  PERMANENT_REDIRECT,
-  BAD_REQUEST,
-  UNAUTHORIZED,
-  PAYMENT_REQUIRED,
-  FORBIDDEN,
-  NOT_FOUND,
-  METHOD_NOT_ALLOWED,
-  NOT_ACCEPTABLE,
-  PROXY_AUTHENTICATION_REQUIRED,
-  REQUEST_TIMEOUT,
-  CONFLICT,
-  GONE,
-  LENGTH_REQUIRED,
-  PRECONDITION_FAILED,
-  PAYLOAD_TOO_LARGE,
-  URI_TOO_LONG,
-  UNSUPPORTED_MEDIA_TYPE,
-  RANGE_NOT_SATISFIABLE,
-  EXPECTATION_FAILED,
-  IM_A_TEAPOT,
-  MISDIRECTED_REQUEST,
-  UNPROCESSABLE_ENTITY,
-  LOCKED,
-  FAILED_DEPENDENCY,
-  TOO_EARLY,
-  UPGRADE_REQUIRED,
-  PRECONDITION_REQUIRED,
-  TOO_MAY_REQUESTS,
-  REQUEST_HEADER_FIELDS_TOO_LARGE,
-  UNAVAILABLE_FOR_LEGAL_REASONS,
-  INTERNAL_SERVER_ERROR,
-  NOT_IMPLEMENTED,
-  BAD_GATEWAY,
-  SERVICE_UNAVAILABLE,
-  GATEWAY_TIMEOUT,
-  HTTP_VERSION_NOT_SUPPORTED,
-  VARIANT_ALSO_NEGOTIATES,
-  INSUFFICIENT_STORAGE,
-  LOOP_DETECTED,
-  NOT_EXTENDED,
-  NETWORK_AUTHENTICATION_REQUIRED,
+/**
+ * A small wrapper around `fetch()` for consuming REST+JSON-APIs.
+ *
+ * @example
+ * ```typescript
+ * import {
+ *   get,
+ *   STATUS_CODE,
+ * } from "https://deno.land/x/olaven/kall/mod.ts";
+ *
+ *  * const { status, todo } = await get(
+ *   "https://jsonplaceholder.typicode.com/todos/1",
+ * );
+ *
+ * console.log(
+ *   status === STATUS_CODE.OK
+ *     ? `Fetched Todo: ${todo}`
+ *     : `${status} when fetching todo..`,
+ * );
+ * ```
+ * @module
+ */
 
-} from "./source/codes.ts";
+export { del, get, patch, post, put } from "./src/methods.ts";
+export { STATUS_CODE } from "./src/codes.ts";
```

Comment:
I believe I like the idea of this change, but I'm not sure about the upper/snake case for `STATUS_CODE`, as it's not a constant value. In my mind it would make more sense if it was `status` or maybe `Status`. I'm leaning towards the former, as it matches the functions exported from `methods.ts`.

@timharek do you have any opinion here?

Reply:
But isn't HTTP status codes constant values? I think we actually can swap these out with Deno's STD: 
https://deno.land/std@0.221.0/http/status.ts

Reply:
I see that I was a bit unclear. What I meant was that `STATUS_CODE` itself is not a constant primitive value, but refers to an object. However, I was not aware of https://deno.land/std@0.221.0/http/status.ts. And they implement it exactly like you've done, it seems :) 

 >  I think we actually can swap these out with Deno's STD

Yes. Let's stick with the standard when there is one :)

Reply:
Ah, I understand now! If it's OK now, you can merge when you see fit and I'll update https://github.com/olaven/NRSS/pull/25 thereafter 😊